src/main/java/lotto/lotto/Lottos.java

```diff
@@ -5,35 +5,59 @@
 import lotto.result.Rank;
 import lotto.result.WinningNumbers;
 
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 import static java.util.stream.Collectors.counting;
 import static java.util.stream.Collectors.groupingBy;
 
 public class Lottos {
+    public static final String EXCEEDED_MANUAL_LOTTOS_MESSAGE = "수동으로 구매한 로또 개수가 총 구입 개수를 초과했습니다.";
+    public static final String NULL_OR_EMPTY_MANUAL_LOTTO_NUMBERS_MESSAGE = "수동으로 구매할 로또의 번호를 입력해주세요.";
+
     private static final int LOTTO_PRICE = 1000;
+    public static final int START_VALUE = 0;
+
     private final List<Lotto> values;
 
     private Lottos(List<Lotto> values) {
         this.values = values;
     }
 
+    public static Lottos of(int purchaseAmount, Lottos manualLottos) {
+        int autoLottosPurchaseAmount = calculateRemainingPurchaseAmount(purchaseAmount, manualLottos);
+        Lottos autoLottos = from(autoLottosPurchaseAmount);
+        return manualLottos.add(autoLottos);
+    }
+
     public static Lottos from(List<Lotto> values) {
         return new Lottos(values);
     }
 
+    public static Lottos from(String[] inputs) {
+        if (isNullOrEmpty(inputs)) {
+            throw new IllegalArgumentException(NULL_OR_EMPTY_MANUAL_LOTTO_NUMBERS_MESSAGE);
+        }
+
+        List<Lotto> lottos = Arrays.stream(inputs)
+                .map(Lotto::from)
+                .collect(Collectors.toList());
+
+        return new Lottos(lottos);
```

Comment:
```suggestion
        return  Arrays.stream(inputs)
                .map(Lotto::from)
                .collect(collectingAndThen(toList(), Lottos::from));
```
이런식으로 사용할 수도 있을 것 같네요 ㅎㅎ